server/src/main/java/org/hl7/davinci/endpoint/controllers/DataController.java

```diff
@@ -156,4 +163,46 @@ public RedirectView reload(@RequestParam String vsac_api_key) {
     return new RedirectView(newUrl);
   }
 
+  public void updateComplianceBundleStatus(String uid) {
+    try {
+      TimeUnit.MINUTES.sleep(1);
+    }
+    catch(Exception e)
+    {
+        System.out.println(e);
+      }
+    Rems rems = remsRepository.findById(uid).orElseThrow(() -> new EntityNotFoundException(uid));
+    rems.setStatus("Approved");
+    remsRepository.save(rems);
+  }
+
+  public void updateComplianceBundleStatusInBackground (final String uid) {
+    Thread t = new Thread(() -> updateComplianceBundleStatus(uid));
+    t.start();
+  }
+
+  @PostMapping(value = "/api/rems")
+  @CrossOrigin
+  public ResponseEntity<Object> postRems(@RequestBody String jsonData) {
+    Gson gson = new GsonBuilder().create();
+    JsonParser parser = new JsonParser();
+    JsonObject remsObject = parser.parse(jsonData).getAsJsonObject();
+    String id = UUID.randomUUID().toString().replace("-", "");
+
+    Rems complianceBundle = new Rems();
+    complianceBundle.setCase_number(id);
+    complianceBundle.setJSON(remsObject.toString());
+    complianceBundle.setStatus("Pending");
+    remsRepository.save(complianceBundle);
+    updateComplianceBundleStatusInBackground(id);
```

Comment:
the status field is internal to the REMS admin, it is not sent via DTR and always setting it to be pending on the initial request makes sense so that no one can set it as approved without the request first being reviewed